### ISSUE TITLE
gimme_taxa.py: sqlite3.IntegrityError

### ISSUE THREAD

**Issue by nick-youngblut** [kblin/ncbi-genome-download#135]
```
$ TAXIDS=1035817,1160721,1297617,1300227,1307,1308,1328,1379690,1384484,1655645,1680,1681,1683,1796613,205913,216816,226186,239935,246787,28025,28116,295405,349741,367928,420247,435590,435591,479437,484020,500634,515619,515620,518635,547043,562,566552,585394,679935,696216,702459,709991,754252,817,818,880074,883062,890402,997877
$ python gimme_taxa.py $TAXIDS  -u -j -o taxids_output.txt
Downloading taxdump.tar.gz from NCBI FTP site (via HTTP)...
Done. Parsing...
Loading node names...
2276414 names loaded.
229123 synonyms loaded.
Loading nodes...
2276414 nodes loaded.
Linking nodes...
Tree is loaded.
Updating database: /ebio/abt3/nyoungblut/.etetoolkit/taxa.sqlite ...
 2276000 generating entries...
Uploading to /ebio/abt3/nyoungblut/.etetoolkit/taxa.sqlite

Inserting synonyms:      95000 Traceback (most recent call last):
  File "bin/scripts/gimme_taxa.py", line 190, in <module>
    main()
  File "bin/scripts/gimme_taxa.py", line 162, in main
    ncbi.update_taxonomy_database()
  File "/ebio/abt3_projects/software/dev/llmga-find-refs/.snakemake/conda/94b17409/lib/python3.6/site-packages/ete3/ncbi_taxonomy/ncbiquery.py", line 129, in update_taxonomy_database
    update_db(self.dbfile)
  File "/ebio/abt3_projects/software/dev/llmga-find-refs/.snakemake/conda/94b17409/lib/python3.6/site-packages/ete3/ncbi_taxonomy/ncbiquery.py", line 760, in update_db
    upload_data(dbfile)
  File "/ebio/abt3_projects/software/dev/llmga-find-refs/.snakemake/conda/94b17409/lib/python3.6/site-packages/ete3/ncbi_taxonomy/ncbiquery.py", line 802, in upload_data
    db.execute("INSERT INTO synonym (taxid, spname) VALUES (?, ?);", (taxid, spname))
sqlite3.IntegrityError: UNIQUE constraint failed: synonym.spname, synonym.taxid
```

**conda env**

```
# Name                    Version                   Build  Channel
_libgcc_mutex             0.1                 conda_forge    conda-forge
_openmp_mutex             4.5                       1_gnu    conda-forge
brotlipy                  0.7.0           py36h8c4c3a4_1000    conda-forge
ca-certificates           2020.6.20            hecda079_0    conda-forge
certifi                   2020.6.20        py36h9f0ad1d_0    conda-forge
cffi                      1.14.3           py36h0ff685e_0    conda-forge
chardet                   3.0.4           py36h9f0ad1d_1007    conda-forge
cryptography              3.1              py36h45558ae_0    conda-forge
dbus                      1.13.6               he372182_0    conda-forge
enum34                    1.1.10           py36h9f0ad1d_1    conda-forge
ete3                      3.1.1                    py36_0    bioconda
expat                     2.2.9                he1b5a44_2    conda-forge
fontconfig                2.13.1            h1056068_1002    conda-forge
freetype                  2.10.2               he06d7ca_0    conda-forge
gettext                   0.19.8.1          hc5be6a0_1002    conda-forge
glib                      2.66.0               h0dae87d_0    conda-forge
gst-plugins-base          1.14.5               h0935bb2_2    conda-forge
gstreamer                 1.14.5               h36ae1b5_2    conda-forge
icu                       67.1                 he1b5a44_0    conda-forge
idna                      2.10               pyh9f0ad1d_0    conda-forge
jpeg                      9d                   h516909a_0    conda-forge
krb5                      1.17.1               hfafb76e_3    conda-forge
ld_impl_linux-64          2.35                 h769bd43_9    conda-forge
libblas                   3.8.0               17_openblas    conda-forge
libcblas                  3.8.0               17_openblas    conda-forge
libclang                  10.0.1          default_hde54327_1    conda-forge
libedit                   3.1.20191231         he28a2e2_2    conda-forge
libevent                  2.1.10               hcdb4288_2    conda-forge
libffi                    3.2.1             he1b5a44_1007    conda-forge
libgcc                    7.2.0                h69d50b8_2    conda-forge
libgcc-ng                 9.3.0               h24d8f2e_16    conda-forge
libgfortran-ng            7.5.0               hdf63c60_16    conda-forge
libgomp                   9.3.0               h24d8f2e_16    conda-forge
libiconv                  1.16                 h516909a_0    conda-forge
liblapack                 3.8.0               17_openblas    conda-forge
libllvm10                 10.0.1               he513fc3_3    conda-forge
libopenblas               0.3.10          pthreads_hb3c22a3_4    conda-forge
libpng                    1.6.37               hed695b0_2    conda-forge
libpq                     12.3                 h5513abc_0    conda-forge
libstdcxx-ng              9.3.0               hdf63c60_16    conda-forge
libuuid                   2.32.1            h14c3975_1000    conda-forge
libxcb                    1.13              h14c3975_1002    conda-forge
libxkbcommon              0.10.0               he1b5a44_0    conda-forge
libxml2                   2.9.10               h68273f3_2    conda-forge
libxslt                   1.1.33               h572872d_1    conda-forge
lxml                      4.5.2            py36h17c4326_0    conda-forge
lz4-c                     1.9.2                he1b5a44_3    conda-forge
mysql-common              8.0.21                        2    conda-forge
mysql-libs                8.0.21               hf3661c5_2    conda-forge
ncbi-genome-download      0.2.6                      py_1    bioconda
ncurses                   6.2                  he1b5a44_1    conda-forge
nspr                      4.28                 he1b5a44_0    conda-forge
nss                       3.57                 he751ad9_0    conda-forge
numpy                     1.19.1           py36h3849536_2    conda-forge
openssl                   1.1.1g               h516909a_1    conda-forge
pcre                      8.44                 he1b5a44_0    conda-forge
pigz                      2.3.4                hed695b0_1    conda-forge
pip                       20.2.3                     py_0    conda-forge
pthread-stubs             0.4               h14c3975_1001    conda-forge
pycparser                 2.20               pyh9f0ad1d_2    conda-forge
pyopenssl                 19.1.0                     py_1    conda-forge
pyqt                      5.12.3           py36haa643ae_3    conda-forge
pyqt5-sip                 4.19.18                  pypi_0    pypi
pyqtchart                 5.12                     pypi_0    pypi
pyqtwebengine             5.12.1                   pypi_0    pypi
pysocks                   1.7.1            py36h9f0ad1d_1    conda-forge
python                    3.6.11          h4d41432_2_cpython    conda-forge
python_abi                3.6                     1_cp36m    conda-forge
qt                        5.12.9               h1f2b2cb_0    conda-forge
readline                  8.0                  he28a2e2_2    conda-forge
requests                  2.24.0             pyh9f0ad1d_0    conda-forge
scipy                     1.5.2            py36h3a855aa_0    conda-forge
setuptools                49.6.0           py36h9f0ad1d_1    conda-forge
six                       1.15.0             pyh9f0ad1d_0    conda-forge
sqlite                    3.33.0               h4cf870e_0    conda-forge
tk                        8.6.10               hed695b0_0    conda-forge
urllib3                   1.25.10                    py_0    conda-forge
wheel                     0.35.1             pyh9f0ad1d_0    conda-forge
xorg-kbproto              1.0.7             h14c3975_1002    conda-forge
xorg-libice               1.0.10               h516909a_0    conda-forge
xorg-libsm                1.2.3             h84519dc_1000    conda-forge
xorg-libx11               1.6.12               h516909a_0    conda-forge
xorg-libxau               1.0.9                h14c3975_0    conda-forge
xorg-libxdmcp             1.1.3                h516909a_0    conda-forge
xorg-libxext              1.3.4                h516909a_0    conda-forge
xorg-libxrender           0.9.10            h516909a_1002    conda-forge
xorg-renderproto          0.11.1            h14c3975_1002    conda-forge
xorg-xextproto            7.3.0             h14c3975_1002    conda-forge
xorg-xproto               7.0.31            h14c3975_1007    conda-forge
xz                        5.2.5                h516909a_1    conda-forge
zlib                      1.2.11            h516909a_1009    conda-forge
zstd                      1.4.5                h6597ccf_2    conda-forge
```

**Comment by nick-youngblut** [comment]
Actually, this is fixed with `ete3 v3.1.2`

**Comment by jrjhealey** [comment]
Maybe totally unrelated but I have been hearing a lot of accounts of people having issues with NCBI in the last couple of days.
